filament/src/components/TransformManager.cpp

```diff
@@ -69,6 +69,28 @@ void FTransformManager::setParent(Instance i, Instance parent) noexcept {
     }
 }
 
+Entity FTransformManager::getParent(Instance i) const noexcept {
+    i = mManager[i].parent;
+    return mManager.getEntity(i);
+}
+
+size_t FTransformManager::getChildCount(Instance i) const noexcept {
+    size_t count = 0;
+    for (Instance ci = mManager[i].firstChild; ci; ci = mManager[ci].next, ++count);
+    return count;
+}
+
+size_t FTransformManager::getChildren(Instance i, utils::Entity* children,
+        size_t count) const noexcept {
+    Instance ci = mManager[i].firstChild;
+    size_t retval = 0;
```

Comment:
maybe `retval` could be named better -- it's the number of children in the list right?

Reply:
Renamed.